constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,104 @@
+mod MemoryChipletAir
+
+### Helper functions/evaluators ###################################################################
+
+fn delta(v: scalar) -> scalar:
+    return v' - v
+
+
+# Enforces that the provided columns must be zero.
+ev is_zero(main: [values[4]]):
+    enf v = 0 for v in values
+
+
+# Enforces that values in the columns in the current row must be equal to the values in the next 
+# row.
+ev equal_seq_lines(main: [values[4]]):
```

Comment:
I still think maybe we should probably pick a better name here. To me something like `is_unchanged` seems slightly better. It would probably be a good idea to think of a good name for evaluator like this since we will probably have constraints like these very commonly.

Reply:
Oops, forgot to change it. Yes, `is_unchanged` seems much better, I'll fix it right away.